packages/clay-css/src/scss/cadmin/variables/_alerts.scss

```diff
@@ -433,6 +438,7 @@ $cadmin-alert-indicator-start: map-deep-merge(
 							padding-left
 						)}
 				),
+			margin-top: 2px,
```

Comment:
I think I ended up missing something in the Inline Alert interaction. In the component implemented in React, for the most common case we have the markup:

```html
<div class="alert alert-info" role="alert">
	<div class="alert-autofit-row autofit-row">
		<div class="autofit-col">
			<div class="autofit-section">
				<span class="alert-indicator">
					<svg class="lexicon-icon lexicon-icon-info-circle" role="presentation" viewBox="0 0 512 512">
						<path class="lexicon-icon-outline" d="M437,75C388.7,26.6,324.4,0,256,0C187.6,0,123.3,26.6,75,75C26.6,123.3,0,187.6,0,256c0,68.4,26.6,132.7,75,181c48.4,48.4,112.6,75,181,75c68.4,0,132.7-26.6,181-75c48.4-48.4,75-112.6,75-181C512,187.6,485.4,123.3,437,75z M288,384c0,17.7-14.3,32-32,32c-17.7,0-32-14.3-32-32V224c0-17.7,14.3-32,32-32c17.7,0,32,14.3,32,32V384z M256,160c-17.7,0-32-14.3-32-32c0-17.7,14.3-32,32-32s32,14.3,32,32C288,145.7,273.7,160,256,160z"></path>
					</svg>
				</span>
			</div>
		</div>
		<div class="autofit-col autofit-col-expand">
			<div class="autofit-section">
				<strong class="lead">Info</strong>
				This is a default alert
			</div>
		</div>
	</div>
</div>
```

But in the examples only of markup, we have:

```html
<div class="alert alert-info" role="alert">
	<span class="alert-indicator">
		<svg class="lexicon-icon lexicon-icon-info-circle" focusable="false" role="presentation" viewBox="0 0 512 512">
			<path class="lexicon-icon-outline" d="M437,75C388.7,26.6,324.4,0,256,0C187.6,0,123.3,26.6,75,75C26.6,123.3,0,187.6,0,256c0,68.4,26.6,132.7,75,181c48.4,48.4,112.6,75,181,75c68.4,0,132.7-26.6,181-75c48.4-48.4,75-112.6,75-181C512,187.6,485.4,123.3,437,75z M288,384c0,17.7-14.3,32-32,32c-17.7,0-32-14.3-32-32V224c0-17.7,14.3-32,32-32c17.7,0,32,14.3,32,32V384z M256,160c-17.7,0-32-14.3-32-32c0-17.7,14.3-32,32-32s32,14.3,32,32C288,145.7,273.7,160,256,160z"></path>
		</svg>
	</span>
	<strong class="lead">Error:</strong> This is an info message
</div>
```

Maybe it's always been that way, but in components we use `ContentRow` and `ContentCol` by default to separate the indicator from the text, this change will fix the indicator's alignment with the text when both are together but will misalign when are not together, in that case, should we keep the text and indicator always together in the same column?

Reply:
This change actually has no effect on what we already had, apparently, it's always been that way.

Reply:
The `ContentRow` and `ContentCol` pattern is for creating the white space between icon and text in `alert-notification`. 
![alert-notification](https://user-images.githubusercontent.com/788266/136827959-f95f42cb-3d5e-4a0c-b310-bc5c8262b0c3.jpg)
I tried to make the icon + text work using different markup configurations since we have a tendency to change things up over time. Inline / Inline Block items are also tricky to vertically align if they have varying font sizes and are wrapped in containers like we do in `alert-notification`. It is also difficult if icon + text + link aren't wrapped in an html element like `span`. For example:

```html
<span class="alert-indicator">
  <svg class="lexicon-icon lexicon-icon-info-circle"></svg>
</span>
<strong class="lead">Info</strong>
This is a
<a href="#1" class="alert-link">info alert</a>
.
```

The text "This is a" isn't wrapped in a `span` tag. I have no way (maybe there is a way I just don't know it) of applying `vertical-align: middle` to it. If we apply `vertical-align` to all the other elements, the text "This is a" won't be correctly aligned.
![info-alert](https://user-images.githubusercontent.com/788266/136828056-336e5273-24b6-4682-9a77-80709e035d69.jpg)

Reply:
I can do that. I'll create an issue for documenting `alert-indicator` vertical alignment.